### ISSUE TITLE
[Luis] 5th scenario: Remove self-doubt and consider removing online dictionary

### ISSUE THREAD

**Issue by julierawe** [w3c/wai-people-use-web-videos#212]
**[HIGH PRIORITY]**
1. Replace self-doubt/negative stereotyping (“I’m not sure I can do it”) with something empowering, like “I know I can do it.” 

2. Remove the reference to “online dictionary” in both audio and visual. Or if EO wants to keep the dictionary, be sure to add that Luis is often frustrated when he does not understand what the definitions mean.

**[MEDIUM PRIORITY]**
1. Consider replacing jargony or difficult words for Luis, such as “difficult," “the meaning of the words,” and “the text is…” 

2. Instead of saying “I have trouble with difficult words,” we suggest framing in the positive: “It helps when job websites show me how to do things using words I know and pictures.”

**COGA's suggested rewording for 5th scenario:**
"I work part-time at a library, sorting books. [DELETE: I’d] [ADD: I would] like to work full-time in an office job [DELETE: but I’m not sure I can do it. I have trouble with difficult words and with math I use an online dictionary and calculator to help me but it takes time. Sometimes it is not just the meaning of the words but the text is hard to follow. I get lost when the text is too long.] [ADD: I know I can figure out how to do it. But it is hard to look online for jobs. It helps when job websites show me how to do things using words I know and pictures. If I get stuck, I look for someone who can help me.] [or visual, show Luis looking for “Help” section or “Contact us.”] But that does not always help. [For visual, show Luis looking at a long page of “Help” text.]" 

In case helpful, here is COGA's consolidated feedback on the Luis script with tracked changes:
https://docs.google.com/document/d/1VzYUBM192J_pvuEcHAsdIywh7xngcbFYR2IaWhiQCLk/edit?usp=sharing

**Comment by nitedog** [comment]
Incorporated much of this wording and concepts with minor editorial changes.
